magenta/magenta/models/nsynth/ours/train.py

```diff
@@ -127,7 +130,7 @@ def main(unused_argv=None):
         global_step=global_step,
         log_every_n_steps=FLAGS.log_period,
         local_init_op=local_init_op,
-        save_interval_secs=300,
+        save_interval_secs=1200,
```

Comment:
If we're going to change this, then how about making it info a command line argument while we're at it?